src/com/qualityworkscg/tests/ServicesTest.java

```diff
@@ -0,0 +1,44 @@
+package com.qualityworkscg.tests;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ServicesTest extends AbstractTest{
+	
+	@Test
+	public void verifyNavigationMobileAutomation () throws InterruptedException{
+		page.clickServices();
+		Thread.sleep(1000);
+		page.clickMobileAutomation();
+		Thread.sleep(2000);
+		Assert.assertEquals(page.getTitle(), "QualityWorks Consulting Group | Mobile and Web Automation", "Testing if title is QualityWorks Consulting Group | Mobile and Web Automation");
+	}
+	
+	@Test
+	public void verifyNavigationDevOpsConsultancy () throws InterruptedException{
+		page.clickServices();
+		Thread.sleep(1000);
```

Comment:
@jpottinger  Is there a way can test these without explicitly calling `Thread.sleep`